app/src/main/java/com/example/someapplication/MoviesListAdapterViewHolder.kt

```diff
@@ -0,0 +1,189 @@
+package com.example.someapplication
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.RecyclerView
+
+class MoviesListAdapterViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+    var title = itemView.findViewById<TextView>(R.id.tv_title_card)
+    var starOne = itemView.findViewById<ImageView>(R.id.iv_star_one_card)
+    var starTwo = itemView.findViewById<ImageView>(R.id.iv_star_two_card)
+    var starThree = itemView.findViewById<ImageView>(R.id.iv_star_three_card)
+    var starFour = itemView.findViewById<ImageView>(R.id.iv_star_four_card)
+    var starFive = itemView.findViewById<ImageView>(R.id.iv_star_five_card)
+    var genre = itemView.findViewById<TextView>(R.id.tv_genre)
+    var reviews = itemView.findViewById<TextView>(R.id.tv_reviews_card)
+    var poster = itemView.findViewById<ImageView>(R.id.iv_poster_card)
+    var time = itemView.findViewById<TextView>(R.id.tv_time_card)
+    var like = itemView.findViewById<ImageView>(R.id.iv_like_card)
+    var ageRate = itemView.findViewById<TextView>(R.id.tv_age_card)
+
+    fun setRate(rate: Int) {
+        when (rate) {
+            1 -> {
+                starOne.setImageDrawable(
```

Comment:
Try to refactor this part of the code using the principle DRY. Use only one iteration by each view.